static/links.less

```diff
@@ -1,10 +1,19 @@
 @import "ui-variables";
 
 a {
-  cursor: pointer;
   color: @text-color-highlight;
   &:hover {
     color: @text-color-highlight;
     text-decoration: underline;
   }
+
+  &[href^="http"],
```

Comment:
@simurai Do you have an idea why grunt-lesslint is failing on this line? This is valid Less, so I'm kinda confused here :confused: